content/en/docs/tasks/administer-cluster/static-pod.md

```diff
@@ -89,7 +89,8 @@ static-web-my-node1        1/1       Running   0          2m
 
 Labels from the static pod are propagated into the mirror-pod and can be used as usual for filtering.
 
-Notice we cannot delete the pod with the API server (e.g. via [`kubectl`](/docs/user-guide/kubectl/) command), kubelet simply won't remove it.
+Note that we should make sure kubelet have enough permissions to create the mirror pod in API server, otherwise the creation request may be forbidden by API server(e.g. [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)).
```

Comment:
Nit: Missing space (" ") before the opening "(", it should be "API server (e.g....)"

Reply:
Done.